### DISCUSSION TITLE
Swap orientation (vertical strip and horizontal workspaces)

### DISCUSSION THREAD

**Discussion by pacien** [YaLTeR/niri#450] [Q&A] [answered]
Hi!

I'm using a multi-monitor setup with vertical (portrait) displays placed side-by-side.

I'd like to try niri, but I don't see in the documentation whether it would be possible to have tiles scrolling vertically instead of horizontally.

Is this kind of setup supported?

**Comment by YaLTeR** [comment, marked as answer]
Hey, it's not supported unfortunately and it would likely be quite annoying to implement. So maybe in the future sometime

**Comment by salman-farooq-sh** [comment]
Possible solution for portrait setups though not exactly the original idea: https://github.com/YaLTeR/niri/discussions/458

**Comment by garrettwp** [comment]
This would be a nice option to have implemented. Hyprland's Hyprscroller plugin has the ability to have windows both horizontal and vertical for scrolling. I use the vertical scrolling for grouped applications or similar windows like terminals or browser windows.

**Comment by tv42** [comment]
Go +1 at https://github.com/YaLTeR/niri/issues/1071
